src/auth.js

```diff
@@ -23,6 +23,7 @@ import { parseJSON } from './util.js'
 
 const apiTokenKey = 'DW-API-KEY'
 const codeVerifierKey = 'DW-CODE-VERIFIER'
+const oAuthBaseSite = process.env.REACT_APP_AUTH_SITE || 'https://auth.data.world'
```

Comment:
Presumably `REACT_APP_AUTH_SITE` is being set in the container environment at some point. Did you find where it's being set? I worry that this will simply be overridden by `https://data.world` when deployed to production.

Reply:
Shouldn't this set of part of config variable in Heroku. I can't confirm this as I don't have access to tableau-connector app Heroku.